packages/template-retail-react-app/app/components/links-list/index.jsx

```diff
@@ -45,11 +45,27 @@ const LinksList = ({
                 ))}
 
             {links && (
-                <List spacing={5} {...styles.list}>
+                <>
                     {variant === 'horizontal' ? (
-                        <HStack>
+                        <>
```

Comment:
The reason we got a11y error was because we wrap li inside a <HStack>. <li> should go directly under <ol> or <ul> as per a11y compliance